src/patternfly/components/ProgressStepper/progress-stepper.scss

```
@@ -264,6 +358,16 @@
 // Step main
 .pf-c-progress-stepper__step-main {
   margin: var(--pf-c-progress-stepper__step-main--MarginTop) var(--pf-c-progress-stepper__step-main--MarginRight) var(--pf-c-progress-stepper__step-main--MarginBottom) var(--pf-c-progress-stepper__step-main--MarginLeft);
+
+  // Draw a new border for vertical alignment using step main
+  .pf-c-progress-stepper__step:not(:last-of-type) > &::before {
+    position: absolute;
+    top: calc(100% + var(--pf-c-progress-stepper__step-main--MarginTop));
+    left: calc(50% - var(--pf-c-progress-stepper__step-connector--before--BorderRightWidth));
```

This is a bit off-center. Part of it is because of the left margin on step-main, but then we only want to move it back by half of its width.

```suggestion
    left: calc(50% - calc(var(--pf-c-progress-stepper__step-connector--before--BorderRightWidth) / 2))
```